src/encoded/static/components/facets/biochemical_inputs.js

```diff
@@ -0,0 +1,204 @@
+import PropTypes from 'prop-types';
+import FacetRegistry from './registry';
+import QueryString from '../../libs/query_string';
+
+
+/**
+ * Biochemical input facet works just like a regular facet but negating is not allowed, links look more like buttons, and there is an "Any" button to select any possible terms
+ */
+const BiochemicalInputsFacet = ({ results, facet, queryString }, reactContext) => {
+    // Get query terms from url
+    const originalQuery = new QueryString(queryString);
+    const originalInputs = originalQuery.getKeyValues('advancedQuery');
+
+    // Possible query entries must be hard-coded because the facet object does not always include all terms, plus we want to exclude rDHS
+    const facetTerms = ['CTCF', 'DNase-seq', 'H3K27ac', 'H3K4me3'];
+
+    // URl should indicate toggle position, but if not, default is "only"
+    let switchState = 'only';
+    if (reactContext.location_href.indexOf('#or') > -1) {
+        switchState = 'or';
+    }
+    // If no toggle position is indicated, "or" is the position when an "AND NOT" is present or when search is not biochemical_inputs:*
+    if (originalInputs && originalInputs[0]) {
```

Comment:
`getKeyValues` always returns an array, which might contain nothing. So `originalInputs` should always exist, though it might contain an empty array. Not that it makes much difference; just a technicality.